pyang/translators/yang.py

```diff
@@ -215,6 +215,7 @@ def emit_stmt(ctx, stmt, fd, level, prev_kwd, prev_kwd_class, islast,
                                indent, indentstep, max_line_len, line_len)
         elif '\n' in stmt.arg:
             # the arg string contains newlines; print it as double quoted
+            line_len -= 1 + len(eol)
```

Comment:
Maybe it would be better just to change the `emit_arg()` argument rather than changing `line_len` (this would more obviously limit the impact of the change)? Please let me know if you'd like me to change this.

Reply:
I'd like your suggestion, which is same like the below else if case `line_len-2`

Reply:
Done.